locales/templates/app.pot

```diff
@@ -229,3 +234,136 @@ msgctxt "firstrun_footer"
 msgid "To be clear, sites and service you use can still know where you've been."
 msgstr ""
 
+msgctxt "your_rights"
+msgid "Your Rights"
+msgstr ""
+
+#. This string is shown when the user has clicked a link that needs to be
+#. opened in a different app.          Argument 2 is the external app name,
+#. argument 1 is our name (i.e. Firefox Focus, or Firefox Klar).
+#, c-format
+msgctxt "external_app_prompt"
+msgid "This link will open in %2$s. Are you sure you want to exit %1$s?"
+msgstr ""
+
+msgctxt "external_app_prompt_no_app_title"
+msgid "No app to handle link"
+msgstr ""
+
+#, c-format
+msgctxt "external_app_prompt_no_app"
+msgid ""
+"None of the apps on your device are able to open this link. Do you want "
+"to exit %1$s and open %2$s to find a suitable app?"
+msgstr ""
+
+#. This label is shown above a list of apps that can be used to open a given
+#. link
+msgctxt "external_multiple_apps_matched_exit"
+msgid "Exit Private Browsing?"
+msgstr ""
+
+#. The page html title (i.e. the <title> tag content)
+msgctxt "errorpage_title"
+msgid "Problem loading page"
+msgstr ""
+
+msgctxt "errorpage_refresh"
+msgid "Try Again"
+msgstr ""
+
+msgctxt "error_connectionfailure_title"
+msgid "Unable to connect"
+msgstr ""
+
+msgctxt "error_connectionfailure_message"
+msgid ""
+"<ul>\n"
+"        <li>The site could be temporarily unavailable or too busy. Try "
+"again in a few moments.</li>\n"
+"        <li>If you are unable to load any pages, check your mobile "
+"device’s data or Wi-Fi connection.</li>\n"
+"      </ul>"
+msgstr ""
+
+msgctxt "error_timeout_title"
+msgid "The connection timed out"
+msgstr ""
+
+msgctxt "error_hostLookup_title"
+msgid "Server not found"
+msgstr ""
+
+msgctxt "error_hostLookup_message"
+msgid ""
+"<ul>\n"
+"        <li>Check the address for typing errors such as\n"
```

Comment:
Those strings should be a 1:1 import from Fennec's netError.dtd (minus the forms). That's how those strings look like in our Android world:
https://github.com/mozilla-mobile/focus-android/blob/master/app/src/main/res/values/strings.xml#L138

Looking at the existing string "teaser" (which also includes a line break): Those show up in our tools like Pontoon like regular line breaks, right?